### PR TITLE
[patch] - install BAS on ROKS

### PR DESCRIPTION
**Pull request by andrercm**

### CODE REVIEW

File: docs/playbooks/bas.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # BAS Playbooks
 
 ## Install BAS
```
Comment (stonepd) [inline docs/playbooks/bas.md:3]: Could you add a description of what the playbook actually does, maybe a summary of the README.md: `Installs **IBM Behavior Analytics Services** on IBM Cloud Openshift Clusters (ROKS) and generates configuration that can be directly applied to IBM Maximo Application Suite.`

File: ibm/mas_devops/roles/bas_install/templates/bascfg.yml.j2 [hunk 2]
```diff
@@ -30,6 +30,42 @@ spec:
...
       crt: |
-        {{bas_tls_crt | indent(8) }}
\ No newline at end of file
+        {{bas_tls_crt[0] | indent(8) }}
+    - alias: part2
+      crt: |
+        {{bas_tls_crt[1] | indent(8) }}
+    - alias: part3
+      crt: |
+        -----BEGIN CERTIFICATE-----
```
Comment (stonepd) [inline ibm/mas_devops/roles/bas_install/templates/bascfg.yml.j2:41]: Is it OK to include this certificate in our code? I see that is has come from an existing public source: [isrgrootx1.pem](https://letsencrypt.org/certs/isrgrootx1.pem) but can you confirm that we are able to include this in our code or should we dynamically include it at script run-time?

**Review by stonepd — CHANGES_REQUESTED** [review]
Technically it looks good, can you address the review comments please

**Comment by andrercm** [comment]
> I want to make sure its ok to include the tls cert in the public github. Is it ok?

@racree i would think it should be ok as the cert i added to make it work is publicly available:
https://letsencrypt.org/certs/isrgrootx1.pem

File: ibm/mas_devops/roles/bas_install/templates/bascfg.yml.j2 (hunk 2 above)
Comment (andrercm) [reply ibm/mas_devops/roles/bas_install/templates/bascfg.yml.j2:41]: i would think it should be okay as it is publicly available: https://letsencrypt.org/certs/isrgrootx1.pem
whether we want to hardcode it or include it via another way... I don't know.
@durera @CAIOFCP any concerns with this approach?

**Comment by andrercm** [comment]
@stonepd responded your points, let me know if you think we should change something else

**Review by stonepd — APPROVED** [review]

**Pull request merged**